src/tests/Common/CLRTest.Execute.targets

```diff
@@ -78,6 +78,7 @@ This file contains the logic for providing Execution Script generation.
 
   <Import Project="CLRTest.Jit.targets" />
   <Import Project="CLRTest.CrossGen.targets" />
+  <Import Project="CLRTest.NativeAot.targets" />
```

Comment:
Nit - we may want to check somewhere that RunCrossgen2 and RunNativeAOT aren't set at the same time, otherwise I suspect interesting things may happen.